apps/src/applab/designProperties.jsx

```diff
@@ -47,16 +59,113 @@ var DesignProperties = module.exports = React.createClass({
         handleDelete={this.props.onDelete}/>);
     }
 
+    var tabHeight = 35;
+    var borderColor = '#c6cacd';
+    var bgColor = '#e7e8ea';
+
+    var defaultTabStyle = {
+      borderColor: borderColor,
+      borderStyle: 'solid',
+      borderWidth: '1px 1px 1px 0',
```

Comment:
apologies for not scanning this in yesterday, but you can peek at the yellow-highlighted diagram face up on my desk for an outline of the strategy used to put borders in the right places here.